addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -52,23 +52,25 @@ _addons = _addons select {_x find "ace_" == 0};
 ///////////////
 // check dlls
 ///////////////
-{
-    private _versionEx = _x callExtension "version";
+if (productVersion select 6 == "Windows") then {
```

Comment:
I don't like the check for "Windows" here.
It would be Better to inverse it and check for the equivalent for Linux/Mac.